python/paddle/tensor/manipulation.py

```diff
@@ -3734,7 +3734,7 @@ def reshape(x, shape, name=None):
         - 3. Given a 3-D tensor x with a shape [2, 4, 6], and the target shape is [-1, 0, 3, 2], the reshape operator will transform x into a 4-D tensor with shape [2, 4, 3, 2] and leaving x's data unchanged. In this case, besides -1, 0 means the actual dimension value is going to be copied from the corresponding dimension of x.
 
     Args:
-        x (Tensor): An N-D Tensor. The data type is ``float32``, ``float64``, ``int32``, ``int64`` or ``bool``
+        x (Tensor): An N-D Tensor. The data type is ``float16``, ``float32``, ``float64``, ``int16``, ``int32``, ``int64``, ``uint16``, ``int8``, ``uint8``, ``complex64``, ``complex128``, ``bfloat16`` or ``bool``.
```

Comment:
no official support for `uint16`, should delete `uint16`?

Reply:
paddle use `bfloat16` to hold `uint16`, so no official support for `uint16`?

``` python
In [42]: x = paddle.to_tensor(123, dtype='uint16')

In [43]: x
Out[43]: 
Tensor(shape=[], dtype=bfloat16, place=Place(cpu), stop_gradient=True,
       123.)
```